mne/simulation/source.py

```diff
@@ -336,28 +340,39 @@ class SourceSimulator(object):
         The number of time samples of the simulation.
     """
 
-    def __init__(self, src, tstep=1e-3, duration=None):
+    def __init__(self, src, tstep=1e-3, duration=None, first_samp=0):
+        if duration is not None and duration < tstep:
+            raise ValueError('duration must be None or >= tstep.')
+        self.first_samp = first_samp
```

Comment:
I would do:
```
self.first_samp = _ensure_int(first_samp, 'first_samp')
```
using `_ensure_int` from `..utils`. It will ensure this is an integer and raise an error if it's not.